src/api/java/mekanism/api/gear/ModuleData.java

```diff
@@ -86,11 +87,18 @@ public final int getMaxStackSize() {
     }
 
     /**
-     * Exclusive modules only work one-at-a-time; when one is enabled, others will be automatically disabled.
+     * Exclusive modules only work one-at-a-time; when one is enabled, incompatible others will be automatically disabled.
      *
      * @return {@code true} if this module type is exclusive.
      */
-    public final boolean isExclusive() {
+    public final boolean isExclusive(int flags) {
```

Comment:
I could actually see just having the no param call, but it does make the calls less explicit so if you don't like that we don't have to and we could set it deprecated. Or it can wait I guess, just let me know which you prefer.

Reply:
Might as well just deprecate it in case anyone ends up making 1.18 addons that add custom modules that they would find this of use.